portfolio/pom.xml

```diff
@@ -12,7 +12,7 @@
     <maven.compiler.source>11</maven.compiler.source>
     <maven.compiler.target>11</maven.compiler.target>
     <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
-    <jetty.version>11.0.0</jetty.version>
+    <jetty.version>9.4.31.v20200723</jetty.version>
```

Comment:
I believe there are jetty versions which should also be updated in the examples in the walkthroughs directory as well. I'm not sure the impacts of not updating those, but it might be a good idea.

Reply:
It looks like you have addressed this in a follow-on PR. I think it might be good practice to try to instead update this PR with the changes that were left out. If this is too complicated, we can skip it, but this is generally how you would respond to review feedback at work. When you get feedback or discover something was left out, you would generally update the existing PR incorporating that feedback.

Reply:
I'm going to mark this resolved so that you can merge these changes and move on to week 2 work.